configuration.yaml

```diff
@@ -48,3 +48,15 @@ git_services:
     - devops/penumbra
     - devops/factory2-jobs
     - rcm/container-image-groups
+  - type: gitlab
+    host: https://gitlab.infra.prod.eng.rdu2.redhat.com
+    token: ENV.GITLAB_SYSOPS_TOKEN
+    repos:
```

Comment:
Could you please add `estuary-playbooks` and `mbs-playbooks`?

Reply:
Added!